.github/workflows/benchmarks.yml

```diff
@@ -1,28 +1,32 @@
 ---
-    name: benchmark tests
-    
-    on:
-      workflow_dispatch:
-    
-    jobs:
-      build:
-        name: Benchmark Tests
-        runs-on: ubuntu-latest
-        steps:
-          - uses: actions/checkout@v4
-    
-          - name: Set up Rust
-            uses: ./.github/actions/setup-rust
-            with:
-              os: 'linux'
-    
-          - name: Set up Spice.ai API Key
-            run: |
-              mkdir -p ~/.spice
-              echo "[spiceai]" > ~/.spice/auth
-              echo "  [spiceai.params]" >> ~/.spice/auth
-              echo '    key = "${{ secrets.SPICE_SECRET_SPICEAI_BENCHMARK_KEY }}"' >> ~/.spice/auth
-    
-          - run: cargo bench -p runtime
-            env:
-                UPLOAD_RESULTS_DATASET: "spiceai.tests.oss_benchmarks"
\ No newline at end of file
+name: benchmark tests
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Benchmark Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Rust
+        uses: ./.github/actions/setup-rust
+        with:
+          os: 'linux'
+
+      - name: Set up Spice.ai API Key
+        run: |
+          mkdir -p ~/.spice
+          echo "[spiceai]" > ~/.spice/auth
+          echo "  [spiceai.params]" >> ~/.spice/auth
+          echo '    key = "${{ secrets.SPICE_SECRET_SPICEAI_BENCHMARK_KEY }}"' >> ~/.spice/auth
+
+      - run: cargo bench -p runtime
+        env:
+          UPLOAD_RESULTS_DATASET: 'spiceai.tests.oss_benchmarks'
+          PG_BENCHMARK_PG_HOST: ${{ secrets.SPICE_SECRET_PG_BENCHMARK_PG_HOST }}
```

Comment:
`PG_BENCHMARK_PG_HOST` - Please check if PG_ prefix is still required as there is ongoing work to simplify naming. We should align to new naming so we don't need to rework this

Reply:
Will confirm and update this after the PG_ prefix changes is included into trunk

Reply:
Confirmed that PG prefix is still needed in spice pod configuration, will keep it as it is